hooks/useWatchConnectivity.ios.ts

```
@@ -139,12 +128,11 @@
           reply({});
         }
       } catch (error) {
-        console.debug('Error handling message:', error);
+        console.error('Error handling message:', error);
         reply({});
       }
     },
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    [fetchWalletTransactions, saveToDisk, wallets],
+    [fetchWalletTransactions, saveToDisk, wallets, constructWalletsToSendToWatch, handleLightningInvoiceCreateRequest]
```

Biome

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)